.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- Before creating an issue please make sure you are using the latest version. -->
+
+<!-- If this is a feature request use this portion of the template and delete the bug portion -->
+**Describe the *feature* request**
+
+
+**What is motivation or use case for changing this?**
+
+
+**Please mention other relevant information such as the browser version, Node.js version, Lighthouse version, and Operating System.**
```

Comment:
we can probably just ask where they want the feature (CLI/DevTools/Extension/Node Module)